overlord/configstate/configcore/corecfg_test.go

```diff
@@ -96,6 +98,11 @@ func (cfg *mockConf) GetPristineMaybe(snapName, key string, result interface{})
 	return nil
 }
 
+func (cfg *mockConf) GetNoVirtual(snapName, key string, result interface{}) error {
```

Comment:
it's a bit unclear whether this the right kind of mocking for this,  shows my point about not adding this until we have callers that needs it for real

Reply:
Yeah, this is now removed